package.json

```diff
@@ -11,6 +11,9 @@
     "lint:js": "next lint",
     "lint:scss": "stylelint '**/*.scss'"
   },
+  "engines" : {
+    "node": ">=16"
+  },
```

Comment:
If you run `npm install`, this will update the `engines` field in `package-lock.json`. Better to commit that change now than to have that file change unexpectedly for whoever pulls this next.

Reply:
Done and done!